frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/utils.ts

```diff
@@ -42,7 +63,7 @@ export const getCephHealthState: ResourceHealthHandler<WatchCephResource> = ({ c
   if (data.length === 0) {
     return { state: HealthState.NOT_AVAILABLE };
   }
-  return CephHealthStatus(status, t);
+  return CephHealthStatus(status, alerts, t);
```

Comment:
We already have ceph object data in manin componnet. We dont need to build health checke here. 
`  const [data, loaded, loadError] = useK8sWatchResource<K8sResourceKind[]>(cephClusterResource);`

1.I would suggest to create a new component for health check
2. Pass ceph details data to taht component
3. Calculate and prepare the results to render